Makefile

```diff
@@ -16,6 +16,7 @@ targets = \
   dbshell \
   deploy \
   dev \
+  fetchremotedb \
```

Comment:
I'm not loving "fetchremotedb".  I would rather see "fetch-remote-db".  I can see how you got to smashing it all together from "dbshell" and "dbload".  The only reason I used "dbshell" is because that's the Django standard.  I think we should switch using '-' separators everywhere and make them more readable 'db-shell', 'load-db', and 'fetch-remote-db'.